lib/cli/error.spec.js

```diff
@@ -22,25 +25,25 @@ describe('error', () => {
 
   it('logs generic message and exists with 1 when called without error', () => {
     logErrorAndExit();
-    expect(console.error.mock.calls[0][0]).toMatchSnapshot();
+    expect(stripAnsi(console.error.mock.calls[0][0])).toMatchSnapshot();
```

Comment:
We can use a [custom serializer](https://github.com/joaogranado/jest-serializer-ansi) here